README.md

```diff
@@ -68,25 +68,27 @@ These steps are to be completed by the author of the blog post.
+   ~~Request a build of the [draft openliberty.io site](https://draft-openlibertyio.mybluemix.net/blog/):~~
+    1. ~~Sign in to [Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) with your GitHub account.~~
+    2. ~~Click **More Options > Trigger Build**. Type `draft` in the **Branch** field, then click **Trigger custom build**.~~
     
-          The draft site build starts running.
+          ~~The draft site build starts running.~~
 
-6. When the build is finished, check that the blog renders correctly on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
+6. When the build is finished, check that the blog renders correctly on the [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/).  In general, use the former (since it will be available much quicker), unless you need to verify links to other content on the site outside of /blogs/.
 
-   If you see any problems (e.g. formatting or typos), resolve them first in your branch, create another PR into `draft` branch (link the PR to the issue again), get the PR merged, then run the [draft site build from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io/branches) again.
+   If you see any problems (e.g. formatting or typos), resolve them first in your branch, create another PR into `draft` branch (link the PR to the issue again), get the PR merged, and wait for IBM Cloud to rebuild [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/), which you can use to verify the change.
 
 7. When you're happy with the post, create a PR from your branch (_not_ from the `draft` branch) to the `staging` branch.
 
    Link the PR to the issue.
-
-   In the PR, provide a link to your post on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/). And, ideally, take a screenshot of the whole post so that reviewers can see what they reviewed after you've made changes.
+ 
+   In the PR, provide a link to your post on the [blog-draft site](https://blog-draft-openlibertyio.mybluemix.net/blog/) or [draft site](https://draft-openlibertyio.mybluemix.net/blog/). And, ideally, take a screenshot of the whole post so that reviewers can see what they reviewed after you've made changes.
```

Comment:
providing a screenshot in the PR seems weird to me, why wouldn't they just click through to draft? But I know this part isn't new so if that's the agreed-upon procedure, np.  Still, it reads strangely- is this the intention:

`Ideally, take a screenshot of the whole post so that reviewers can see what the post in context as it will appear on the site.`